src/Factories/FunFunFactory.php

```diff
@@ -1642,7 +1632,7 @@ public function newIsCustomDonationAmountValidator(): IsCustomAmountValidator {
 	}
 
 	public function getSkinDirectory(): string {
-		return 'skins/' . $this->getSkinSettings()->getSkin() . '/templates';
+		return $this->getChoiceFactory()->getSkinForUserBucket();
```

Comment:
👍 

I like this wrapper! This ensures that when the skin campaign is over and 10h16 finally dead, this method can be reverted to building a string.